index.js

```diff
@@ -3,7 +3,6 @@ import {
   getTasksFromStorage, saveTasksToStorage, /* editTaskDescription, deleteTask, */
 } from './modules/taskModule.js';
 import renderTasks from './modules/renderModule.js';
-// import updateTaskStatus from './modules/statusModule.js'; // Import the new module
```

Comment:
I think it would be better to remove the comments from your code, its against coding standards and it will make your code dry